From 66297217838be972f24c6f034475b633348e6f03 Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Sun, 24 Aug 2025 11:28:58 +0200
Subject: [PATCH 039/116] net: pcs: try to fix unmet depency

kismet warnings: (new ones prefixed by >>)
>> kismet: WARNING: unmet direct dependencies detected for FWNODE_PCS when selected by PCS_MTK_USXGMII
   WARNING: unmet direct dependencies detected for FWNODE_PCS
     Depends on [n]: NETDEVICES [=y] && (ACPI [=n] || OF [=n]) && PHYLINK [=y]
     Selected by [y]:
     - PCS_MTK_USXGMII [=y] && NETDEVICES [=y]

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202508241628.eq7tx9wU-lkp@intel.com/
---
 drivers/net/pcs/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/pcs/Kconfig b/drivers/net/pcs/Kconfig
index b804def5bdd8..dde5c7b0a7aa 100644
--- a/drivers/net/pcs/Kconfig
+++ b/drivers/net/pcs/Kconfig
@@ -41,9 +41,9 @@ config PCS_MTK_LYNXI
 
 config PCS_MTK_USXGMII
 	tristate "MediaTek USXGMII PCS"
+	select PHYLINK
 	select FWNODE_PCS
 	select PCS_MTK_LYNXI
-	select PHYLINK
 	imply PHY_MTK_PEXTP
 	help
 	  This module provides a driver for MediaTek's USXGMII PCS supporting
-- 
2.39.5

